nemo_skills/conversion/nemo_to_hf.py

```diff
@@ -128,15 +128,18 @@ def convert(input_nemo_path, output_hf_path, hf_model_name, max_shard_size, prec
     heads_per_group = head_num // num_query_groups
     qkv_total_dim = head_num + 2 * num_query_groups
 
+    weight_base_name = 'model'
+    if f'{weight_base_name}.embedding.word_embeddings.weight' not in model.state_dict():
+        weight_base_name = 'model.module'
```

Comment:
For which models is this necessary? I would ideally avoid changing this script much since this is mostly a copy-paste from nemo and if we add new things here, it would be harder to merge upstream changes later